deploy/bundle/catalogsource.yaml

```diff
@@ -0,0 +1,262 @@
+                            - name: OPERATOR_NAME
+                              value: windows-machine-config-operator
+                      nodeSelector:
+                        beta.kubernetes.io/os: linux
+                      serviceAccountName: windows-machine-config-operator
+        installModes:
+          - type: OwnNamespace
+            supported: true
+          - type: SingleNamespace
+            supported: true
+          - type: MultiNamespace
+            supported: false
+          - type: AllNamespaces
+            supported: false
+  customResourceDefinitions: |
+    - apiVersion: apiextensions.k8s.io/v1beta1
+      kind: CustomResourceDefinition
+      metadata:
+        name: windowsmachineconfigs.wmc.openshift.io
+      spec:
+        group: wmc.openshift.io
+        names:
+          kind: WindowsMachineConfig
+          listKind: WindowsMachineConfigList
+          plural: windowsmachineconfigs
+          singular: windowsmachineconfig
+        scope: Namespaced
+        subresources:
+          status: {}
+        validation:
+          openAPIV3Schema:
+            description: WindowsMachineConfig is the Schema for the windowsmachineconfigs API
+            properties:
+              apiVersion:
+                description: >-
+                  APIVersion defines the versioned schema of this representation of an
+                  object. Servers should convert recognized schemas to the latest
+                  internal value, and may reject unrecognized values. More info:
+                  https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources
+                type: string
+              kind:
+                description: >-
+                  Kind is a string value representing the REST resource this object
+                  represents. Servers may infer this from the endpoint the client
+                  submits requests to. Cannot be updated. In CamelCase. More info:
+                  https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
+                type: string
+              metadata:
+                type: object
+              spec:
+                description: >-
+                  WindowsMachineConfigSpec defines the desired state of
+                  WindowsMachineConfig
+                type: object
+              status:
+                description: >-
+                  WindowsMachineConfigStatus defines the observed state of
+                  WindowsMachineConfig
+                type: object
+            type: object
+        version: v1alpha1
+        versions:
+          - name: v1alpha1
+            served: true
+            storage: true
+  packages: |
+    - channels:
+      - currentCSV: windows-machine-config-operator.v0.0.1
+        name: alpha
+      defaultChannel: "alpha"
+      packageName: windows-machine-config-operator
+kind: ConfigMap
+metadata:
+  name: wmco-operators
```

Comment:
why this name?

Reply:
Its an arbitrary name. Whats wrong with it?

Reply:
Makes sense, I'll change it to wmc-operators